website/src/components/SequenceDetailsPage/DataTableEntryValue.tsx

```diff
@@ -70,4 +78,13 @@ const PlainValueDisplay: React.FC<{ value: TableDataEntry['value'] }> = ({ value
     return <span className='italic'>None</span>;
 };
 
+const generateCleanHtml = (trustedHtml: string, userValue: string): string => {
+    const cleanedValue = sanitizeHtml(userValue, {
```

Comment:
Could you explain the behaviour here and how it helps with your goal? When I read this it sounds like it doesn't allow any tags? I don't see how that will let you display a table (except one with only one variable cell)?

Reply:
Or will you have some complicated javascript in the HTML?

Reply:
My plan was to use a custom element. The `html`-config would be set to something like `<frequency-custom-display>__value__</frequency-custom-display>` and through `additionalHeadHTML`, I'd include a JS file that registers the element. Here's a prototype:

[example.zip](https://github.com/user-attachments/files/18551143/example.zip)

Reply:
Cool - my only feedback is I feel like`html` is a confusing name. To me it implies that one can supply HTML. Something like `wrappedInHtml` or `htmlTemplate` would be a better name IMO for the `type`.

Reply:
I like `htmlTemplate`, just renamed it.

Reply:
I would also guess that your solution might yield weird behaviour with hydration? Idk if it's worth testing it before merging in case we're adding something that's not of any use?

Reply:
(maybe you have!)

Reply:
Hmm, good question, haven't thought about that – yes, might be good to give it a try first.

Reply:
(I edited my comment as I felt less confident but yeah, I am bouncing back and forth so maybe worth checking)

Reply:
![image](https://github.com/user-attachments/assets/e3962b17-b949-43d2-bf75-b6a037b11f3f)

It works!

Reply:
Did you check the console says nothing about hydration?

Reply:
Just checked - can't see a message - great!